src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -340,6 +340,15 @@ private static ImmutableArray<IVariableDeclaration> GetVariableDeclarationStatem
                         OperationFactory.CreateVariableDeclaration(declaration.LocalSymbol, Create(declaration.InitializerOpt), declaration.Syntax)));
         }
 
+        private static readonly ConditionalWeakTable<BoundTupleExpression, object> s_tupleElementsMappings =
```

Comment:
The problem with CWT is that it has a 'class' constraint on the values.  But we're storing ImmutableArrays.  So we end up needing to box things into object to satisfy the CWT.